rdmo/management/assets/js/store/configureStore.js

```diff
@@ -28,7 +39,6 @@ export default function configureStore() {
   // load: restore the config from the local storage
   const updateConfigFromLocalStorage = () => {
     const ls = {...localStorage}
-
     Object.entries(ls).forEach(([lsPath, lsValue]) => {
```

Comment:
Can you add `if (lsPath.startsWith('rdmo.management.config.')) {` after this line. This was an oversight by me. Otherwise `rdmo.storeid` and everything else in the store is picked up as well and stored again unter e.g. `rdmo.management.config.rdmo.storeid`.